test/tools/WebListener/Controllers/ResponseController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Extensions;
+using Microsoft.Extensions.Primitives;
+using mvc.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace mvc.Controllers
+{
+    public class ResponseController : Controller
+    {
+        public String Index()
+        {
+            
+            string output = String.Empty;
+            string contentType = Constants.ApplicationJson;
+
+            StringValues contentTypes;
+            if (Request.Query.TryGetValue("contenttype", out contentTypes))
+            {
+                contentType = contentTypes.FirstOrDefault();
+            }
+
+            StringValues statusCodes;
+            Int32 statusCode;
+            if (
+                Request.Query.TryGetValue("statuscode", out statusCodes) && 
+                Int32.TryParse(statusCodes.FirstOrDefault(), out statusCode)
+            )
```

Comment:
We never use the formatting.

Reply:
Do i really need to have this all on one line where you have to scroll for an eternity to see it?

Reply:
I believe we use the pattern:
```c#
if (Request.Query.TryGetValue("statuscode", out statusCodes) &&   
    Int32.TryParse(statusCodes.FirstOrDefault(), out statusCode))  
{
    ...
}
```

Reply:
fixed